wordpress-mobile/WordPress-iOS
# Jetpack Extensions: Use different keys to avoid conflicts between WP and JP extensions

Fixes #19591

## Description
This PR fixes an issue where jetpack extensions were conflicting with WP extensions. This is done by using different keys for the WP and JP extensions.

P.S: This change means we will have to migrate the extensions' data since they are no longer shared between WP and JP. This will be done in a coming PR and will follow the same pattern as the one in #19663, that's why this PR depends on #19663's branch.

## Testing Instructions

1. Install & run Jetpack
2. Sign in using account A
3. Install & run WordPress
4. Sign in using account B
5. Go to photos
6. Select a photo
7. Tap on the share button
8. Select Jetpack
9. Make sure that the site list displayed is the one from account A, not account B.


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
